styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1 +1,87 @@
-
+a.footer-menu-link {
+  color: darken($light-gray, 15%);
+  font-size: 1.2rem;
+}
+a.footer-tab-link{
+  color:$white;
+  font-size: 1.3rem;
+  text-transform: uppercase;
+  &:hover{
+    cursor: default;
+    text-decoration: none;
+  }
+}
+ul.footer-menu{
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+ul.footer-tabs{
+  list-style: none;
+  padding: 0;
+  margin-bottom: 25px;
+}
+ul.footer-menu-split {
+    column-count: 3;
+}
+li.footer-menu-item {
+  list-style: none;
+  line-height: 25px;
+}
+@media only screen and (max-width: $screen-sm) {
+  a.footer-menu-link{
+    display: block;
+    background-color: lighten($blue-gray, 05%);
```

Comment:
just blue-gray man. no lighten for the bg color por favor. it wont match the other elements.